src/Router.tsx

```diff
@@ -3,6 +3,7 @@ import { BrowserRouter, Switch, Route } from 'react-router-dom';
 import HomePage from './Home/HomePage';
 import VerificationLandingPage from './Login/VerificationLandingPage';
 import NavWrapper from './NavWrapper/NavWrapper';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
We should comment out instead of ignoring the error

Reply:
@PanopticaRising said that if a file isn't included it throws errors which is why it is here
We should leave a comment for this
Better yet we could just do `import './Adviser/AdviserPage';`